### PR TITLE
Update measure-page.md

### PR DESCRIPTION
**Pull request by spencerball**
Updates to Data sources, Methodology, and Contact details sections.

### CODE REVIEW

File: measure-page.md [hunk 1]
```diff
@@ -57,27 +57,65 @@ If other, please contact the team to arrange for another option to be added.
...
 The department responsible for analysing and/or publishing the data (for example, Department for Education).
 
 ### Source URL
 
-Provide a link to the data source.
+Provide a link to the data source used for the measure. Users should be able to find the source data easily from the page given as the source URL.
+
+The link should usually be the landing page for the statistics publication, rather than a PDF, spreadsheet or GOV.UK ‘collections’ page.
+
+> For example, for the 2017 Key Stage 1 measures, the correct URL is [Phonics screening check and key stage 1 assessments: England 2017](https://www.gov.uk/government/statistics/phonics-screening-check-and-key-stage-1-assessments-england-2017), not [Statistics: key stage 1](https://www.gov.uk/government/collections/statistics-key-stage-1) (collections page) or the [statistical first release PDF](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/654859/Phonics_KS1_SFR_Text_2017_.pdf)..
```
Comment (TheDoubleK) [inline measure-page.md]: There are two full stops at the end here.

File: measure-page.md [hunk 2]
```diff
@@ -143,26 +168,22 @@ Use this section to explain factors that might lead users to misinterpret or mis
...
 * the source of the data (for example, ‘The figures are drawn from the 2014/15 and 2015/16 English Housing Survey (EHS)’)
-* technical aspects that need further explanation (for example, ‘Some households contain people from different ethnic backgrounds; in these circumstances, we have used the ethnic background of the household reference person to define the ethnic background of the household’)
+* technical aspects that need further explanation (for example, ‘some households contain people from different ethnic backgrounds; in these circumstances, we have used the ethnic background of the household reference person to define the ethnic background of the household’)
 * plain English definitions of terms used in the measure (for example, ‘Young people’ means people aged 10 to 17 years’)
-* what’s included where ‘other’ is used to categorise certain parts of the data (for example, ‘Types of crime under ‘Other offences’ include:...’)
+* what’s included where ‘other’ is used to categorise certain parts of the data (for example, ‘Types of crime under ‘Other offences’ include:...’
```
Comment (TheDoubleK) [inline measure-page.md]: I think we needed the closing bracket here?  Or we also need to delete the opening bracket.

File: measure-page.md [hunk 3]
```diff
@@ -208,7 +210,7 @@ The language in this section may need to be more specialist and technical than t
 
 Include information about the following if they apply to your data:
 
-* sample size, including limitations on covering the whole population
+* sample size , including limitations on covering the whole population
```
Comment (TheDoubleK) [inline measure-page.md]: Space before a comma?

File: measure-page.md [hunk 4]
```diff
@@ -255,57 +263,72 @@ Please specify what the download file contains, for example:
...
-### Summaries
+### Summary
 
-Use the summary to comment in detail on the findings presented in this dimension.
+Use the summary to comment in detail on the findings presented in this dimension. 
 
 Content should:
 
-* avoid repetition of the main points by providing detail specific to this dimension
+* avoid repetition of the Main points by providing detail specific to this dimension
```
Comment (TheDoubleK) [inline measure-page.md]: Does Main need to be capitalised here?

**Comment by spencerball** [comment]
Thanks @TheDoubleK - great spots. I've made those tweaks as suggested.

**Pull request merged**